### ISSUE TITLE
tm_obs_switcher errors getting configuration from TM Manager 2023_24.1.1

### ISSUE THREAD

**Issue by paulsp** [MayorMonty/tm-obs-switcher#17]
tm_obs_switcher errors getting configuration from TM Manager 2023_24.1.1.  I understand TM is making internal changes which I suspect is the root cause. 

```
tm-obs-switcher v1.5.0
Created by Brendan McGuire (brendan@bren.app)

? VEX TM Address: 127.0.0.1
? VEX TM Password: ****
? Would you like to control OBS? Yes
? OBS Websocket Address: ws://127.0.0.1:4455
? OBS Websocket (leave blank for no password): 
? Would you like to control an ATEM device over the network? No

Connecting to servers...
❌ Tournament Manager: invalid json response body at http://127.0.0.1/fieldsets reason: Unexpected token  in JSON at position 1
exiting...
```

**Comment by MayorMonty** [comment]
Yep, DWAB has changed the internal WebSocket API this project relies on. I haven't had the chance to fully look at the new APi yet, but will try to push out an update that addresses this soon

**Comment by paulsp** [comment]
Brendan,
FYI: Per traffic I have seen on  https://www.vexforum.com <https://www.vexforum.com/>, TournamentManager Is in the middle of some internal changes.

Paul Spencer

**Comment by MayorMonty** [comment]
Fixed by #18

**Comment by paulsp** [comment]
I have confirmed tm-obs-switcher v1.6.2 runs in the following environment
- M2 MacBook Air
- MacOS 13.4.1
- Tournament Manager v2023_24.1.1
- OBS Studio v29.1.3(64 bit)

Thank you for the application and your support!
